docs/ytt-data-values.md

```diff
@@ -1,27 +1,32 @@
 ## ytt @data/values
 
-- [Defining data values](#defining-data-values)
+The standard way to externalize configuration values is to declare them as "Data Values"
+and then reference those values in templates.
+
+- [Defining and Using Data Values](#declaring-and-using-data-values)
 - [Splitting data values into multiple files](#splitting-data-values-into-multiple-files)
 - [Overriding data values via command line flags](#overriding-data-values-via-command-line-flags)
 - [Library data values](#library-data-values)
   - [Setting via files](#library-setting-via-files)
   - [Setting via command line flags](#library-setting-via-cmd)
 
-### Defining data values
+### Declaring and Using Data Values
 
-One way to inject input data into templates is to include a YAML document annotated with `@data/values`. Example:
+A Data Values file is a YAML document annotated with `@data/values`.
 
 ```yaml
 #@data/values
 ---
 key1: val1
 key2:
   nested: val2
-key3:
+key3: val3
 key4:
 ```
 
-Subsequently these values can be accessed via `@ytt:data` library:
+`ytt` processes all Data Values files prior to rendering templates.
+
+Templates access those processed values via the `@ytt:data` module:
```

Comment:
Templates "can" access...
Templates "are able to" access...

Reply:
Thank you! using the tersest form.